### ISSUE TITLE
 inform everyone who uses the fantastic 'hecatomb' related to OSError on Windows' WSL or WSL2

### ISSUE THREAD

**Issue by Key-man-fromArchive** [shandley/hecatomb#90]
When installing Hecatomb in a WSL environment, there are cases where an OSerror occurs during the Megahit step. Even after reinstalling WSL several times, the issue persists and cannot be resolved.

It seems that the OSerror occurs and prevents file writing when MEGAHIT creates a temp folder in the working directory, which may be the cause of the issue.

When creating a folder and moving files using Windows Explorer, the owner and group of the created file or folder are often determined as "Root".

However, Hecatomb process running on WSL operates under the "user id" and only has the permissions of the "user id".

Therefore, it seems that Hecatomb and Megahit, which have the permissions of the "user id", are unable to properly create or handle the temp folder at this moment.

The solution is simple. You can change the owner and group of the working directory to "user id" using the chown command to resolve the issue.

I struggled due to a sudden OSerror issue for three days. Now that I have found the answer, I am sharing this experience. Thank you.

**Comment by beardymcjohnface** [comment]
Thank you very much for this insight. Being able to run on WSL would be very useful for anyone with a beefy windows pc but no convenient access to a linux server. I had not thought to check if it indeed ran on WSL. I'm curious, does this error occur if you `sudo hecatomb run ...` ?

**Comment by Key-man-fromArchive** [comment]
Yes, I have already tried it. When I enter 'sudo hecatomb run', it displays an error message 'command not found'
